keepassxc-browser/content/keepassxc-browser.js

```diff
@@ -1115,19 +1115,28 @@ kpxc.preparePageForMultipleCredentials = function(credentials) {
         return;
     }
 
-    function getLoginText(credential) {
+    function getLoginText(credential, withGroup) {
+        const group = (withGroup && credential.group) ? `[${credential.group}] ` : '';
         const visibleLogin = (credential.login.length > 0) ? credential.login : tr('credentialsNoUsername');
+        const text = `${group}${credential.name} (${visibleLogin})`;
```

Comment:
Was this ordering already fixed? I cannot see @droidmonkey's message about the line anymore.

Reply:
I don't remember any comment regarding this, can you add it again please?

Reply:
While waiting for the answer, I would actually prefer that the group name would be added last. The username should be always visible as first.

Reply:
In my opinion I would rather not add settings because already now there are so many and it's hard to understand all of them. They give the impression that KeepassXC is only for tech people but it wouldn't have to be. Also it adds complexity to the code.

Maybe rather come up with a good way to display these 3 informations in a sane way and do not fear the change. Maybe some people need to get used to it but it won't take long to adopt I believe.

But in the end it is your project. You decide and I will change it :)

Reply:
I agree that there's too much different options already. 

Just a sidenote, but I'm already planning a settings page upgrade and it would definitely need some Advaced settings tab where we could put all extra settings which doesn't concern the normal user.